tests/python/ConfigTest.py

```diff
@@ -863,6 +893,7 @@ def test_virtual_display(self):
 
 
 class ConfigVirtualWithActiveDisplayTest(unittest.TestCase):
+
```

Comment:
it seems useless, you can remove it.

Reply:
I added for coherence of coding style compared to other classes but removed from the commit now to avoid polluting diff.